src/runner/index.js

```diff
@@ -311,6 +312,35 @@ export default class Runner extends EventEmitter {
         this._validateProxyBypassOption();
     }
 
+    _validateTestForAllowMultipleWindowsOption (tests) {
+        if (tests.some(test => test.isLegacy))
+            throw new GeneralError(RUNTIME_ERRORS.cannotUseAllowMultipleWindowsOptionForLegacyTests);
```

Comment:
What if in a bunch of tests only es-next tests need a multi-window functionality?

Reply:
I can report a warning that 'allowMultipleWindows' option is not supported for legacy tests.

Reply:
for me warning is better for this case

Reply:
Ok.

Reply:
Discussed in private (legacy tests could hang)